doc/howto/network_configure.md

```diff
@@ -12,6 +14,19 @@ lxc network set UPLINK dns.nameservers=8.8.8.8
 
 The available configuration options differ depending on the network type.
 See {ref}`network-types` for links to the configuration options for each network type.
+````
+````{group-tab} UI
+To edit the configuration of a network, navigate to the overview page for the network, and observe its attributes and settings..
```

Comment:
I think we should be a bit more detailed here, like "Open networks from the side navigation, then click the network name to open its detail page."